content/02.delete-me.md

```diff
@@ -1,15 +1,165 @@
+### Heading 3
 
-If you notice a problem with Manubot, it's best to submit an upstream fix to the appropriate repository:
-[`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) for the git repository stub or [`greenelab/manubot`](https://github.com/greenelab/manubot) for the Python package.
+#### Heading 4
+
+`Heading 1`'s are recommended to be reserved for the title of the manuscript
+
+`Heading 2`'s are recommended for sections such as *Abstract*, *Methods*, *Conclusion*, etc.
+
+<!-- an arbitrary comment. not visible in output. -->
+
+Horizontal rule:
+
+---
+
+A long paragraph of text.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+Another long paragraph.
+Donec massa sapien faucibus et molestie ac feugiat sed lectus.
+Est sit amet facilisis magna etiam.
+Ornare arcu dui vivamus arcu felis bibendum.
+Magna fermentum iaculis eu non diam.
+Vehicula ipsum a arcu cursus vitae.
+Ante metus dictum at tempor commodo ullamcorper a.
+Vitae tortor condimentum lacinia quis vel eros.
+Ut venenatis tellus in metus vulputate eu scelerisque felis imperdiet.
+Facilisis sed odio morbi quis.
+Enim ut sem viverra aliquet eget sit amet tellus.
+
+*Italic* _text_
+
+**Bold** __text__
+
+Combined *italics and __bold__*
+
+~~Strikethrough~~
+
+Line break without starting a new paragraph by putting  
+two spaces at end of line.
+
+Putting each sentence on its own line has numerous benefits with regard to [editing](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) and [version control](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
+
+1. Ordered list item
+2. Ordered list item
+    a. Sub-item
+    b. Sub-item
+        i. Sub-sub-item
+3. Ordered list item
+    a. Sub-item
+
+- List item
+- List item
+- List item
+
+superscript<sup>2</sup>
+
+subscript<sub>2n+1</sub>
+
+unicode superscript™
+
+unicode subscriptₓ
+
+**Links and Citations**
+
+Bare URL link: <https://manubot.org>
+
+[Link with text](https://www.manubot.org)
+
+[Link with hover text](https://www.manubot.org "Manubot Homepage")
+
+[Link styled as a button](https://www.manubot.org "Manubot Homepage"){.button}
+
+[Link by reference][manubot homepage]
+
+[Manubot Homepage]: https://www.manubot.org
+
+Citation via DOI: [@doi:10.15363/thinklab.4]
+
+Citation by tag: [@tag:deep-review]
+
+Manual URL citation: [@url:https://greenelab.github.io/meta-review]
+
+**Figures and Tables**
+
```

Comment:
Should we add
```
Referencing figures or tables:
```

Reply:
Re: "multiple citations together", I have that near the bottom already:

![image](https://user-images.githubusercontent.com/8326331/53830493-91f1fe80-3f50-11e9-97a6-7d2413214602.png)

Should we move this paragraph up to the references section?

___

Re: "Referencing figures or tables", I had that in there at one point, but that also compelled me to try making sub-headings for all of the other things too, and I had trouble naming them. It kinda looks weird to give only the reference links a section heading and not anything else. If you'd like to brainstorm section names for the other things with me, let me know.

Reply:
> Re: "multiple citations together", I have that near the bottom already:

I dont's see [that here](https://github.com/manubot/rootstock/blob/fbf95c2d8a60ab37abd29eb7573a18ebcc57a6b8/content/02.delete-me.md). I think we probably want to introduce multiple citations in the same place we introduce single citations.

BTW we call the in-text numbers citations and the items in the bibliography references. https://drsaraheaton.wordpress.com/2013/10/18/whats-the-difference-between-a-citation-and-a-reference/ This isn't a hard and fast rule everywhere but we've been generally abiding by that terminology.

Reply:
Sorry, that must've gotten left out of my last push somehow. It's in my master local repo and it will be there in the final push before merge. You can see the current working state (my local repo) reflected in the front-end demo: https://www.vincentrubinetti.com/manubot/manuscript.html Do you want to move that paragraph up?

I'll talk to you about the terminology.